src/contexts/FilterContext.jsx

```diff
@@ -0,0 +1,25 @@
+import { createContext, useContext, useReducer, useState, useEffect } from "react";
+import { filterReducer } from "../reducer/filterReducer";
+
```

Comment:
I don't think context was needed for filters if you are filtering only on single page

Reply:
Thank you, I'll check.